### ISSUE TITLE
Did output for internal requests with API::raw() change in the last version?

### ISSUE THREAD

**Issue by Youbility** [dingo/api#371]
Hey Jason. After the last composer update I experienced an altered behaviour when performing internal requests. It seems like the data does not go through the transformer anymore. The code I use to do the request is the following:

~~~
$jsonData = \API::raw()
               ->get('properties/64', 'include=pictures,publishing')
               ->getContent();
~~~

Before update my results looked like `"interiorquality":"luxury"`; now it looks like `"interiorquality_id":6`. Is the new behaviour going to be the new default? Because what I actually want to work with when doing internal API requests is the already transformed data. Otherwise I could for example use the repository.

I had to change the composer.json to get the old behaviour back. Instead of requiring `"dingo/api": "~0.8.0",` I'm now explicitly requiring `"dingo/api": "0.8.2",`

**Comment by jasonlewis** [comment]
I must've missed this. So sorry.

Hm, looking at the diff it does look like I've tinkered with it. I believe in #256 it was originally decided that the response was still run through the transformer.

Apologies!

**Comment by jasonlewis** [comment]
Fixed in latest `0.8.*@dev`, will tag soon. Cheers.

**Comment by Youbility** [comment]
Thank you so much. Cheers!
